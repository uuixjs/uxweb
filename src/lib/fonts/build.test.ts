import globby from "globby";
import { resolve } from "path";

const FONTS_DIR = resolve(__dirname, "../fonts");

describe("package", () => {
  it("exports the same fonts", () => {
    expect(globby.sync("*", { cwd: FONTS_DIR }).sort()).toMatchSnapshot();
  });
});
