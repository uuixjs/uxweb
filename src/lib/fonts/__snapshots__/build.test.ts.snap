// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`package exports the same fonts 1`] = `
Array [
  "Roobert-Bold.woff",
  "Roobert-Bold.woff2",
  "Roobert-Regular.woff",
  "Roobert-Regular.woff2",
  "Roobert-SemiBold.woff",
  "Roobert-SemiBold.woff2",
  "fonts.json",
]
`;
